src/urllib3/connection.py

```diff
@@ -109,6 +109,9 @@ def __init__(self, *args, **kw):
         #: provided, we use the default options.
         self.socket_options = kw.pop("socket_options", self.default_socket_options)
 
+        # Protocol used to talk to the proxy.
+        self.proxy_scheme = kw.pop("proxy_scheme", None)
```

Comment:
Where is `self.proxy_scheme` used?

Reply:
Good catch, it's been removed. Left over from initial code.